### ISSUE TITLE
Last NVDA alpha - Unable to rename folder of source code

### ISSUE THREAD

**Issue by CyrilleB79** [nvaccess/nvda#12166]
### Context

I have found this issue by copying source code tree from one computer to another. I was used to copy the prepared source code tree from my home's computer to my company's computer since I do not have Visual studio installed on my company's computer.
This was allowing me to fix issues and/or test solutions on my company's computer without Visual Studio, e.g. for software I only have on my company's computer (e.g. Skype for Business, Matlab, et.)

### Steps to reproduce:

* Clone NVDA source code
  By default, it is cloned in a folder named "nvda"
* Prepare the source tree (with `scons source` command)
* Run NVDA executing the new `runnvda.bat` script
  NVDA starts correctly
* Close the NVDA run from source
* Rename the folder containing the clone of NVDA's srouce code, e.g. from "nvda" to "nvda1"
* Run NVDA executing the new `runnvda.bat` script

### Actual behavior:

An error message appears saying:
> NVDA cannot  detect the Python virtual environment. To run NVDA from source, please use runnvda.bat in the root of this repository.

### Expected behavior:

* NVDA starts correctly
* If not possible:
  * at least this unclear message should not appear since I have actually run NVDA with the `runnvda.bat` script.
  * A way to restore a good configuration with the new folder name should be provided

### Additional comments

Being able to copy a prepared source tree may be useful in the following situation:
* Debugging with source code on computer where Visual Studio is not installed
* Be able to save a source tree in another folder rather than committing it or stashing it 

#### NVDA installed/portable/running from source:
From source

#### NVDA version:
Version: source-master-52bf1ed

#### Windows version:
Windows 10 2004 (64-bit) build 19041.388
#### Name and version of other software in use when reproducing the issue:
N/A
#### Other information about your system:
N/A
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Tried some previous post-2020.4 alpha version of NVDA, where the issue was not present.

#### If add-ons are disabled, is your problem still occurring?
Yes
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
No

Issue very likely to be introduced with #12075 in 528d5708aa13a0a384e1fcd5ef00a933ba077695.
Cc @michaelDCurran 

Also Cc @lukaszgo1: if you are interested in looking at this issue.
